src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect"  
+      href="https://fonts.gstatic.com" 
+      crossorigin="*"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:wght@400;700&family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__card card" data-qa="card">
+        <img 
+          class="card__img" 
+          src="./images/imac.jpeg" 
+          alt="APPLE A1419 iMac 27"
+        >
+    
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+    
+        <p class="card__code">Product code: 195434</p>
+    
+        <div class="card__review review">
+          <div class="review__stars review__stars--4">
+            <div class="review__stars-star"></div>
```

Comment:
```suggestion
        <div class="card__review ">
          <div class="card__stars stars stars--4">
            <div class="stars__star"></div>
```
You don't need to make the review a separate block. There should be only 2 block in this task
![image](https://user-images.githubusercontent.com/96409563/226444746-646c5562-91d2-4a2f-bc66-98d264e81357.png)